common/changes/@itwin/property-grid-react/fix-empty-props-toggle-broken-multiselect_2022-08-03-20-00.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/property-grid-react",
+      "comment": "Using disableUnifiedSelection property to get FilteringPropertyGrid to work with multiple elements",
```

Comment:
is this comment relevant now?

Reply:
might also want to update the pr description as necessary

Reply:
good catch, i'll fix that